media-tv/mythtv/mythtv-30.0_p20190808.ebuild

```diff
@@ -0,0 +1,369 @@
+
+	myconf+=("$(use_enable systemd systemd_notify)")
+	myconf+=("$(use_enable systemd systemd_journal)")
+
+	# Bindings
+	if use perl && use python; then
+		myconf+=("--with-bindings=perl,python")
+	elif use perl; then
+		myconf+=("--without-bindings=python")
+		myconf+=("--with-bindings=perl")
+	elif use python; then
+		myconf+=("--without-bindings=perl")
+		myconf+=("--with-bindings=python")
+	else
+		myconf+=("--without-bindings=perl,python")
+	fi
+
+	use python && myconf+=("--python=${EPYTHON}")
+
+	# External codec library options (used for mythffmpeg and streaming transcode)
+	myconf+=("--enable-libmp3lame") # lame is not optional it is required for some broadcasts for silence detection of commercials
+	myconf+=("$(use_enable xvid libxvid)")
+	myconf+=("$(use_enable x264 libx264)")
+	myconf+=("$(use_enable x265 libx265)")
+	myconf+=("$(use_enable vpx libvpx)")
+
+	if use debug; then
+		myconf+=("--compile-type=debug")
+		#myconf+=("--enable-debug") does nothing per sphery
+		myconf+=("--disable-stripping") # FIXME: does not disable for all files, only for some
+		myconf+=("--enable-valgrind") # disables timeouts for valgrind memory debugging
+	else
+		myconf+=("--compile-type=release")
+	fi
+
+	# Video Output Support
+	myconf+=("--enable-x11")
+	myconf+=("--enable-xrandr")
+	myconf+=("--enable-xv")
+	myconf+=("$(use_enable vdpau)")
+	myconf+=("$(use_enable vaapi)")
+
+	# Random stuff
+	myconf+=("$(use_enable zeroconf libdns-sd)")
+	myconf+=("$(use_enable libass)")
+
+	# Clean up DSO load times and other compiler bits
+	myconf+=("--enable-symbol-visibility")
+	myconf+=("--enable-pic")
+
+	if tc-is-cross-compiler ; then
+		myconf+=("--enable-cross-compile --arch=$(tc-arch-kernel)")
+		myconf+=("--cross-prefix=${CHOST}-")
+	fi
+
+	# Build boosters
+	has distcc ${FEATURES} || myconf+=("--disable-distcc")
+	has ccache ${FEATURES} || myconf+=("--disable-ccache")
+
+	einfo "Running ./configure ${myconf[@]}"
+	./configure \
+		--cc="$(tc-getCC)" \
+		--cxx="$(tc-getCXX)" \
+		--ar="$(tc-getAR)" \
+		--extra-cflags="${CFLAGS}" \
+		--extra-cxxflags="${CXXFLAGS}" \
+		--extra-ldflags="${LDFLAGS}" \
+		--qmake=$(qt5_get_bindir)/qmake \
+		${myconf[@]}
+}
+
+src_install() {
+	emake STRIP="true" INSTALL_ROOT="${D}" install
+	dodoc AUTHORS UPGRADING README
+	readme.gentoo_create_doc
+
+	insinto /usr/share/mythtv/database
+	doins database/*
+
+	newinitd "${FILESDIR}"/mythbackend.init-r2 mythbackend
+	newconfd "${FILESDIR}"/mythbackend.conf-r1 mythbackend
+	if use systemd; then
+		systemd_newunit "${FILESDIR}"/mythbackend.service-28 mythbackend.service
+	fi
+
+	dodoc keys.txt
+
+	keepdir /etc/mythtv
+	fowners -R mythtv /etc/mythtv
+	keepdir /var/log/mythtv
+	fowners -R mythtv /var/log/mythtv
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/mythtv.logrotate.d-r4 mythtv
+
+	insinto /usr/share/mythtv/contrib
+	# Ensure we don't install scripts needing the perl bindings (bug #516968)
+	use perl || find contrib/ -name '*.pl' -exec rm -f {} \;
```

Comment:
Open this up and die on find properly.

Reply:
It is OK for **find** to fail if there are no files to remove.  Failure to remove a file with **rm -f** is OK too since it may have been passed a directory by find.